guides/v2.1/extension-dev-guide/cache/partial-caching/database-caching.md

```diff
@@ -130,77 +130,80 @@ To enable database caching using a custom cache frontend, you must modify `<your
 		    ],
 		],
 
-	An example is shown in <a href="#mage-cache-db-config">Configuration examples</a>.
+	An example is shown in [Configuration examples].
 
 4.	Save your changes to `env.php` and exit the text editor.
 5.	Continue with the next section.
 
-<h2 id="mage-cache-db-verify">Verify database caching is working</h2>
+## Verify database caching is working {#mage-cache-db-verify}
 To verify database caching is working, clear the current cache directories, go to any cacheable page in a web browser, and verify that data is written to the database and not to the file system.
 
 Use the following steps:
 
-1.	If you haven't done so already, log in to the Magento server as, or switch to, the <a href="{{ page.baseurl }}/install-gde/prereq/file-sys-perms-over.html">Magento file system owner</a>.
+1.	If you haven't done so already, log in to the Magento server as, or switch to, the [Magento file system owner].
 2.	Clear the current cache directories:
 
-		rm -rf <your Magento install dir>/var/cache/* <your Magento install dir>/var/page_cache/* <your Magento install dir>/var/di/* <your Magento install dir>/var/generation/*
+    ```bash
+		rm -rf <your Magento install dir>/var/cache/* <your Magento install dir>/var/page_cache/* <your Magento install dir>/generated/metadata/* <your Magento install dir>/generated/code/*
+    ```
 
 3.	In a web browser, go to any cacheable page (such as the {% glossarytooltip 1a70d3ac-6bd9-475a-8937-5f80ca785c14 %}storefront{% endglossarytooltip %} front door page).
 
-	If exceptions display, verify `di.xml` syntax and try again. (To see exceptions in the browser, you must <a href="{{ page.baseurl }}/config-guide/cli/config-cli-subcommands-mode.html#config-mode">enable developer mode</a>.)
+	If exceptions display, verify `di.xml` syntax and try again. (To see exceptions in the browser, you must [enable developer mode].)
 4.	Enter the following commands:
 
 		ls <your Magento install dir>/var/cache/*
 		ls <your Magento install dir>/var/page_cache/*
 
-    <div class="bs-callout bs-callout-info" id="info">
-      <span class="glyphicon-class">
-      <p>Due to a known issue, a custom cache frontend still results in some objects being cached to the file system; however, fewer assets are cached compared to file system caching.</p>
-      <p>If you use the <code>default</code> cache frontend, you don't have this issue.</p></span>
-    </div>
+    {:.bs-callout .bs-callout-info #info}
+    <span class="glyphicon-class">
```

Comment:
Remove HTML span element

Reply:
@leviv, I think I made this comment as you were making changes to the callout classes. Please remove span.